commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -100,8 +100,7 @@ def questions(self) -> List[Dict[str, Any]]:
                 "type": "input",
                 "name": "scope",
                 "message": (
-                    "Scope. Could be anything specifying place of the "
-                    "commit change (users, db, poll):\n"
+                    "What is the scope of this change? (class or file name): (press enter to skip)\n"
```

Comment:
I'd wrap enter with quare brakets: `[enter]` to make it clear is a keyboard key, like in linux